src/traces/ohlc/attributes.js

```diff
@@ -0,0 +1,105 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../../lib');
+var scatterAttrs = require('../scatter/attributes');
+
+var lineAttrs = scatterAttrs.line;
+
+var directionAttrs = {
+    visible: {
+        valType: 'enumerated',
+        values: [true, false, 'legendonly'],
+        role: 'info',
+        dflt: true,
+        description: [
+
+        ].join(' ')
+    },
+
+    color: Lib.extendFlat({}, lineAttrs.color),
+    width: Lib.extendFlat({}, lineAttrs.width),
+    dash: Lib.extendFlat({}, lineAttrs.dash),
+
+    tickwidth: {
+        valType: 'number',
+        min: 0,
+        max: 1,
+        dflt: 0.1,
+        role: 'style',
+        description: [
+            'Sets the width of the open/close tick marks',
+            'relative to the *t* minimal interval.'
+        ].join(' ')
+    }
+};
+
+module.exports = {
+
+    // or should this be 'x'
+    //
+    //
+    // should we add the option for ohlc along y-axis ??
+    t: {
```

Comment:
I don't have a very strong opinion on this, but my gut reaction would have been to make it `x`, and change it to `y` if we offered an option for horizontal values like we do with bar charts (though I can't find any examples of people actually doing this). Stela says she thinks she's seen these charts against a different axis (asset type, for example). I can't find an example of this, but can imagine folks wanting to see if different stocks or asset types were subject to the same trends during a given time period (you'd have to normalize their values somehow of course). And If that were to happen, it would be awkward to call stock A, stock B, etc `t` but calling them `x` would be natural.